src/TinyCiv.Shared/Events/Client/MoveUnitEvent.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace TinyCiv.Shared.Events.Client;
+
+//  Requires implementation
+public record MoveUnitEvent(Guid unitId, int newRow, int newColumn) : ClientEvent;
```

Comment:
Since you added names in camelCase, properties of this record will also have them in camelCase, which is against C# naming convention